connectors/automation-anywhere/element-templates/automation-anywhere-connector.json

```diff
@@ -0,0 +1,692 @@
+  },
+  "documentationRef": "https://docs.camunda.io/docs/components/modeler/web-modeler/connectors/available-connectors/automation-anywhere/",
+  "category": {
+    "id": "connectors",
+    "name": "Connectors"
+  },
+  "appliesTo": [
+    "bpmn:Task"
+  ],
+  "elementType": {
+    "value": "bpmn:ServiceTask"
+  },
+  "groups": [
+    {
+      "id": "operation",
+      "label": "Operation"
+    },
+    {
+      "id": "authentication",
+      "label": "Authentication"
+    },
+    {
+      "id": "configuration",
+      "label": "Configuration"
+    },
+    {
+      "id": "input",
+      "label": "Input"
+    },
+    {
+      "id": "output",
+      "label": "Output"
+    },
+    {
+      "id": "errors",
+      "label": "Error Handling"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:http-json:1",
+      "binding": {
+        "type": "zeebe:taskDefinition:type"
+      }
+    },
+    {
+      "label": "Operation type",
+      "id": "operationType",
+      "group": "operation",
+      "value": "crateWorkItemModel",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Create Work Item model",
+          "value": "createWorkItemModel"
+        },
+        {
+          "name": "Create queues",
+          "value": "createQueues"
+        },
+        {
+          "name": "Add Work Items to the queue",
+          "value": "addWorkItemsToTheQueue"
+        },
+        {
+          "name": "Update the Work Item by Id",
+          "value": "updateWorkItem"
+        },
+        {
+          "name": "List workload management queues",
+          "value": "listWorkloadQueues"
+        },
+        {
+          "name": "List of all the available workItemModelId",
+          "value": "listAvailableWorkItemModelId"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operationType"
+      }
+    },
+    {
+      "label": "Type",
+      "id": "authenticationType",
+      "group": "authentication",
+      "description": "Choose the authentication type",
+      "value": "userNamePasswordAuth",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Authenticate (username and password)",
+          "value": "userNamePasswordAuth"
+        },
+        {
+          "name": "Authenticate (username and apiKey)",
+          "value": "userNameApiKeyAuth"
+        },
+        {
+          "name": "Authentication (refresh) token",
+          "value": "tokenAuth"
+        },
+        {
+          "name": "Bearer token",
+          "value": "bearer"
+        }
```

Comment:
Please check namng from HTTP rest connector or UIPath. We don't use word `token` and specify the protocol.